src/chat/tools/getDockerContainers.ts

```diff
@@ -121,6 +120,10 @@ export class GetDockerContainersTool extends BaseLanguageModelTool<IGetDockerCon
   ): Promise<TextOnlyToolResultPart> {
     const parameters = toolCall.input as IGetDockerContainersParameters;
 
+    stream.progress(
+      `Retrieving available Docker containers with parameters: ${JSON.stringify(parameters)}...`,
```

Comment:
Since `resourceKind` is required, we should be able to reference it directly here:
```suggestion
      `Retrieving available ${parameters.resourceKind} Docker containers...`,
```